release/models/platform/openconfig-platform.yang

```diff
@@ -542,6 +548,17 @@ module openconfig-platform {
         device, one as primary and the other as secondary, should both
         report the same value.";
     }
+
+    leaf base-mac-address {
+      type oc-yang:mac-address;
+      description
+        "This is a MAC address representing the root or primary MAC
+        address for a component.  Only components such as CHASSIS and
```

Comment:
If we remove the word "only" this seems to increase the re-usability of the model. As an example Access Point radios also utilize a "base" mac address from which all BSSIDs utilize over-the-air.